src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        ColorSupplier color = new ColorSupplier();
```

Comment:
make those two private class-level fields, to avoid needless recreation per each method call